firmware/lucidgloves-firmware/WIFISerialCommunication.ino

```diff
@@ -0,0 +1,72 @@
+//only compiles if WIFISerial is set because it won't compile for a non-compatible board
+#if COMMUNICATION == COMM_WIFISERIAL
+#include <WiFi.h>
+#include <HTTPClient.h>
+
+class WIFISerialCommunication : public ICommunication {
+
+private:
+  bool m_isOpen;
+  WiFiClient client;
+
+  void connect() {
+    while (!client.connect(host, port)) //blocks thread until connected
+      delay(500);
+    Serial.print("Connected to Client at" + String(client.localIP()) + ":" + String(client.localPort()));
+    client.println("Hello, world");
+
+  }
```

Comment:
This function doesn't seem to be used anywhere? If it's not used, let's remove it.